tokens/usecase/token_registry.go

```diff
@@ -49,6 +49,8 @@ func GetTokensFromChainRegistry(chainRegistryAssetsFileURL string) (map[string]d
 		token.HumanDenom = asset.Symbol
 		token.IsUnlisted = asset.Preview
 		token.CoingeckoID = asset.CoingeckoID
+		token.Name = asset.Name
```

Comment:
What is the name to be used for? Would human denom/symbol not be sufficient?

Reply:
Name of the asset is required by Dexscreener /asset endpoint, e.g.
```
{
"asset": {
"id": "ibc/078AD6F581E8115CDFBD8FFA29D8C71AFE250CE952AFF80040CBC64868D44AD3",
"name": "Tether (Ethereum)",
"symbol": "USDT.eth.pica",
"totalSupply": "5937.672387000000000000000000000000000000",
"coinGeckoId": "tether"
}
}
```